lib/bureaucrat/helpers.ex

```diff
@@ -133,6 +133,13 @@ defmodule Bureaucrat.Helpers do
 
   def format_test_name("test " <> name), do: name
 
+  def format_test_name(function_name),
+    do: raise("It looks like you called a `Phoenix.ConnTest` macro inside `#{function_name}`.
```

Comment:
I'm a huge fan of `do:` but this is not a one-liner expression so I think it would work better as a regular do-end. Also, I suggest using the `"""` format for the message, because in the current version all the lines except for the 1st one will be indented (which I assume is not intended).